src/index.html

```diff
@@ -8,7 +8,224 @@
+              </div>
+              <div>
+                <ul>
+                  <li style="display:inline-block">
+                    <img src="cinema.jpg" alt="icon1">
+                    <p>Cinema</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Sport.jpg" alt="icon1">
+                    <p>Sport</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Theatre.jpg" alt="icon1">
+                    <p>Theatre</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="BBQ.jpg" alt="icon1">
+                    <p>BBQ</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Party.jpg" alt="icon1">
+                    <p>Party</p>
+                  </li>
+                  <li style="display:inline-block">
+                    <img src="Cooking.jpg" alt="icon1">
+                    <p>Cooking</p>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+
+          <!-- Skill -->
+          <div id="skill">
+            <div id="skill_title">
+              <h2 style="text-transform:uppercase">skill</h2>
+            </div>
+            <div id="skill_container">
+              <div id="block1">
+                <h6>Microsoft Excel</h6>
+              </div>
+              <div id="block2">
+                <h6>Microsoft Word</h6>
+              </div>
+              <div id="block3">
+                <h6>Adobe</h6>
+              </div>
+              <div id="block4">
+                <h6>Spanish</h6>
+              </div>
+              <div id="block5">
+                <h6>Powerpoint</h6>
+              </div>
+              <div id="block6">
+                <h6>French</h6>
+              </div>
+            </div>
+          </div>
+          <!--Quick ID-->
+          <div id="quick_id_container">
+            <div id="quick_id_title">
+              <h2 style="text-transform:uppercase">quick id</h2>
+            </div>
+            <div id="ID_Container">
+              <table border="0">
+                <tr>
+                  <td><b>Phone</b></td>
+                  <td>123-456-7890</td>
+                </tr>
+                <tr>
+                  <td><b>Email</b></td>
+                  <td>info@mysite.com</td>
+                </tr>
+                <tr>
+                  <td><b>Website</b></td>
+                  <td>www.wix.com</td>
+                </tr>
+                <tr>
+                  <td><b>Address</b></td>
+                  <td>500 Terry Francois Street<br>
+                      San Francisco, CA 94158</td>
+                </tr>
+              </table>
+            </div>
+          </div>
+
+        </div>
+
+      </body>
+      <!--footer-->
+      <footer>
+        <div style="width:100%">
```

Comment:
Don't think you need this inline style `width:100%` here since `div` is block element.